conf/.env.example

```diff
@@ -16,32 +16,53 @@ APP_DEBUG=true
 
 # This should be your email address
 
-SITE_OWNER=__EMAIL__
+SITE_OWNER=__APP__@__DOMAIN__
 
 
 # The encryption key for your database and sessions. Keep this very secure.
 # If you generate a new one all existing data must be considered LOST.
-# Change it to a string of exactly 32 chars or use command `php artisan key:generate` to generate it
+#
+# You can leave this empty if you use `php artisan 2fauth:install`.
+# Otherwise, change it to a string of exactly 32 chars or use command
+# `php artisan key:generate` to generate it.
 
-APP_KEY=SomeRandomStringOf32CharsExactly
+APP_KEY=__KEY__
 
 
-# This variable must match your installation's external address but keep in mind that
-# it's only used on the command line as a fallback value.
+# This variable must match your installation's external address.
+# Webauthn won't work otherwise.
 
 APP_URL=http://__DOMAIN__
 
 
+# If you want to serve js assets from a CDN (like https://cdn.example.com),
+# uncomment the following line and set this var with the CDN url.
+# Otherwise, let this line commented.
+
+ASSET_URL=/
+
+
+# The domain subdirectory from which you want to serve 2FAuth.
+# This must reflect the path targeted by APP_URL.
+#
+# For example, if you set APP_URL=https://mydomain.org/2fa to access 2FAuth from the '/2fa/' subdirectory
+# you have to set APP_SUBDIRECTORY=2fa
+#
+# Leave blank if you serve 2FAuth from the domain root.
+
+APP_SUBDIRECTORY=
```

Comment:
So we could support sub-path installs now ?

Reply:
I read that recently yes, I still have to enable and test that